### PR TITLE
Improve the low level CRTP documentation 

### PR DESCRIPTION
**Pull request by ataffanel**
This PR improves the low level CRTP documentation.

Documentation pages are created for all ports implemented in the Crazyflie.

### CODE REVIEW

**Comment by ataffanel** [comment]
Github is showing way more commits than I made for this branch. I have rebased the branch a couple of time, I will investigate what could be happening ...

File: docs/functional-areas/crtp/crtp_platform.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+Answer:
+
+| Byte | Description |
+|------|-------------|
+| 0    | getProtocolVersion (0) |
+| 1    | Version |
+
+Returns the CRTP protocol version. See the
+[protocl versioning and stability guarantee](index.md#protocol-version-and-stability-guarantee) documentation for more
```
Comment (knmcguire) [inline docs/functional-areas/crtp/crtp_platform.md:67]: typo: protocol

File: docs/functional-areas/crtp/index.md [hunk 2]
```diff
@@ -3,69 +3,137 @@ title: CRTP - Communication with the Crazyflie
...
-Port allocation
----------------
 
-Current port allocation:
+## Link implementations
+
+There is currently two actively supported link implementation. They are
+documented on their own thread:
+ - [The radio link]() implements CRTP link over nRF24 compatible radios
+ - [The USB link]() implements CRTP link over USB to the Crazylfie 2.x USB port
```
Comment (knmcguire) [inline docs/functional-areas/crtp/index.md:61]: These links now to the same page it is now. A bit confusing as I would expect to either go to a subtitle or a different page.

**Review by knmcguire — COMMENTED** [review]
Just a couple of commnets:)

File: docs/functional-areas/crtp/index.md (hunk 2 above)
Comment (ataffanel) [reply docs/functional-areas/crtp/index.md:61]: Those are not documented yet. I remove the link.

**Comment by ataffanel** [comment]
Thanks for the Review Kimberly. Implemented your suggested fixes.

**Pull request merged**